doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -494,10 +495,23 @@ UICR.SECONDARY.MPCCONF
 
 .. _ug_nrf54h20_ironside_se_programming:
 
-Programming
-***********
+.. _ug_nrf54h20_ironside_se_programming:
```

Comment:
@hakonfam @SebastianBoe Please review this page

Reply:
LGTM